river/sync.go

```diff
@@ -479,3 +462,33 @@ func (r *River) doBulk(reqs []*elastic.BulkRequest) error {
 
 	return nil
 }
+
+// get mysql field value and convert it to specific value to es
+func (r *River) getFieldValue(col *schema.TableColumn, fieldType string, value interface{}) interface{} {
+	var fieldValue interface{}
+	switch fieldType {
+	case fieldTypeList:
+		v := r.makeReqColumnData(col, value)
+		if str, ok := v.(string); ok {
+			fieldValue = strings.Split(str, ",")
+		} else {
+			fieldValue = v
+		}
+
+	case fieldTypeDate:
+		if col.Type == schema.TYPE_NUMBER {
+			col.Type = schema.TYPE_DATETIME
+			switch value := value.(type) {
+			case int64:
+				fieldValue = r.makeReqColumnData(col, time.Unix(value, 0).Format("2006-01-02 15:04:05"))
```

Comment:
use mysql.TimeFormat direclty

Reply:
What's the mysql.TimeFormat mean?

Reply:
`mysql.TimeFormat` is the const name for "2006-01-02 15:04:05"